### PR TITLE
ENG-4283 Add default lang page title validation (v7.1.3)

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </parent>
     <artifactId>entando-plugin-jpseo</artifactId>
     <groupId>org.entando.entando.plugins</groupId>
-    <version>7.1.1</version>
+    <version>7.1.3</version>
```
Comment (zonia3000) [inline pom.xml:11]: I've just seen similar situations (moving from 7.1.0 to 7.1.2, skipping one version) with Breno yesterday. We agreed with Walter that for minor versions we increment the modules without skipping versions, so here you should put 7.1.2
This of course lead to mixed versions in de-app, see for example the de-app 7.1.2 release PR: https://github.com/entando/entando-de-app/pull/448/files

**Review by zonia3000 — APPROVED** [review]

File: src/main/java/org/entando/entando/plugins/jpseo/web/page/SeoPageController.java [hunk 2]
```diff
@@ -127,6 +135,19 @@ private void validatePagePlacement(PageRequest pageRequest, BindingResult bindin
             throw new ValidationUnprocessableEntityException(bindingResult);
         }
     }
+    private void validateDefaultLangPageTitle(PageRequest pageRequest, BindingResult bindingResult) {
+
+        String defaultLangTitle = pageRequest.getTitles().get(langManager.getDefaultLang().getCode());
+
+        if ((null == defaultLangTitle) || (defaultLangTitle.isEmpty())) {
+            String defaultLangCode = langManager.getDefaultLang().getCode().toUpperCase();
+            bindingResult.reject("12", "Invalid title for the default language " + defaultLangCode);
```
Comment (cecchisandrone) [inline src/main/java/org/entando/entando/plugins/jpseo/web/page/SeoPageController.java]: How do we decide this "magic" number? Do we have constants somewhere?

Comment (alepintus) [reply src/main/java/org/entando/entando/plugins/jpseo/web/page/SeoPageController.java]: @cecchisandrone we have a list of error codes here:

https://github.com/entando/entando-engine/blob/714d3a6b449affaa3f4fdc02dbd3966c697e86af/src/main/java/org/entando/entando/web/page/validator/PageValidator.java#L42

and here we have other 2 error codes:

https://github.com/entando/entando-plugin-jpseo/blob/04f08219194552dfed858dc7c2d4e0f947eca8dc/src/main/java/org/entando/entando/plugins/jpseo/web/page/SeoPageController.java#L101

https://github.com/entando/entando-plugin-jpseo/blob/04f08219194552dfed858dc7c2d4e0f947eca8dc/src/main/java/org/entando/entando/plugins/jpseo/web/page/validator/SeoPageValidator.java#L79

Comment (alepintus) [reply src/main/java/org/entando/entando/plugins/jpseo/web/page/SeoPageController.java]: @cecchisandrone as you suggested I moved the error codes to static variables

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**